components/raftstore/src/store/fsm/store.rs

```diff
@@ -655,6 +655,11 @@ impl<T: Transport, C: PdClient> PollHandler<PeerFsm<RocksEngine>, StoreFsm> for
             |_| unreachable!()
         );
 
+        // We must renew current_time because this value may be created a long time ago.
+        // If we do not renew it, this time may be smaller than propose_time of a command,
+        // which was proposed in another thread while this thread receives its AppendEntriesResponse
+        //  and is ready to calculate its commit-log-duration.
+        self.poll_ctx.current_time.replace(monotonic_raw_now());
```

Comment:
How about `self.poll_ctx.current_time = None`?

Reply:
Sorry, I thought `replace` will work for null value.